engine/crates/engine-v2/src/sources/graphql/mod.rs

```diff
@@ -84,6 +84,7 @@ impl<'ctx> GraphqlExecutor<'ctx> {
             })
             .await?
             .bytes;
+        tracing::debug!("{}", String::from_utf8_lossy(&bytes));
```

Comment:
still need this?

Reply:
half the time it's useful I'd say. Not sure, but it has no cost at runtime if not activated. So I'll leave it for now I think.